client/components/data/query-themes/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import shallowEqual from 'react-pure-render/shallowEqual';
```

Comment:
I really have no preference here but I see [56](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=isequal&type=Code) references for [`lodash.isEqual`](https://lodash.com/docs/4.17.2#isEqual) and only [5](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=shallowEqual&type=Code) for `shallowEqual`. I assume that for stuff like this we prefer using the lodash flavor and my guess is that if we're using this so infrequently we might as well trim the fat and get rid of it whenever we can.